al2/aarch64/standard/2.0/Dockerfile

```
@@ -361,7 +361,7 @@ RUN set -ex \
     && docker-compose version
 
 # Node 12
-ENV NODE_12_VERSION="12.18.0"
+ENV NODE_12_VERSION="12.21.0"
```

```suggestion
ENV NODE_12_VERSION="12.22.2"
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/